common-util/api/moduleDetails.ts

```diff
@@ -1,11 +1,19 @@
 import { ContributeModuleDetails } from 'types/moduleDetails';
 
 export const getModuleDetails = async () => {
-  const response = await fetch('/api/moduleDetails');
+  const response = await fetch('/api/module-details', {
+    headers: {
+      /* Prevent caching on the server / by browsers.
+       * Required in order to update the UI with the latest posts on user's action.
+       */
+      'Cache-Control': 'max-age=0, no-cache',
```

Comment:
I think it would make sense to configure globally, we can add vercel.json file in root folder. (can define source as /api) https://vercel.com/docs/edge-cache#using-vercel.json-and-next.config.js 

so that we also don't do it here and inside the server side function

Reply:
yes I think it would make sense, especially considering all of them were fetched by intervals previously